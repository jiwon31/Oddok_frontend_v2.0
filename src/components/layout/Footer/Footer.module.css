.footer {
  width: 100%;
  background-color: black;
  color: var(--gray300);
  padding: 1.5em 1rem;
}

.info {
  display: flex;
  justify-content: space-between;
  align-items: center;
  gap: 1rem;
  padding: 0 7rem;
}

.title {
  flex-basis: 50%;
  display: flex;
  align-items: center;
}

.logo {
  font-size: 1.8rem;
  font-weight: bold;
  margin-right: 0.4em;
}

.name {
  font-size: 1.1rem;
  font-weight: 500;
}

.contents,
.content {
  margin: 0;
}

.contents {
  font-size: 1.2rem;
  font-weight: 500;
}

.copyright {
  padding-top: 2rem;
  padding-bottom: 1rem;
  text-align: center;
}

.copyright > p {
  margin: 0;
}
