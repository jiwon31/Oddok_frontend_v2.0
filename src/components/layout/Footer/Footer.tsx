import styles from "./Footer.module.css";

function Footer() {
  return (
    <footer className={styles.footer}>
      <div className={styles.info}>
        <div className={styles.title}>
          <h2 className={styles.logo}>ODDOK</h2>
          <p className={styles.name}>실시간 화상 스터디 서비스</p>
        </div>
        <div className={styles.contact}>
          <p className={styles.contents}>Contact</p>
          <p className={styles.content}>yejin013@naver.com</p>
        </div>
        <div className={styles.team}>
          <p className={styles.contents}>Team</p>
          <p className={styles.content}>DOBBY @숭실대학교 소프트웨어학부</p>
        </div>
      </div>
      <div className={styles.copyright}>
        <p>Copyright Dobby 2022, All Rights Reserved.</p>
      </div>
    </footer>
  );
}
export default Footer;
