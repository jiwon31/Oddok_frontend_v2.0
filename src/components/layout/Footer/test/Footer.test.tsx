import renderer from "react-test-renderer";
import Footer from "../Footer";

describe("Footer", () => {
  it("renders", () => {
    const component = renderer.create(<Footer />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
