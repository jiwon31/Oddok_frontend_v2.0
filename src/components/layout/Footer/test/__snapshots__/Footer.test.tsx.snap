// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders 1`] = `
<footer
  className="footer"
>
  <div
    className="info"
  >
    <div
      className="title"
    >
      <h2
        className="logo"
      >
        ODDOK
      </h2>
      <p
        className="name"
      >
        실시간 화상 스터디 서비스
      </p>
    </div>
    <div
      className="contact"
    >
      <p
        className="contents"
      >
        Contact
      </p>
      <p
        className="content"
      >
        yejin013@naver.com
      </p>
    </div>
    <div
      className="team"
    >
      <p
        className="contents"
      >
        Team
      </p>
      <p
        className="content"
      >
        DOBBY @숭실대학교 소프트웨어학부
      </p>
    </div>
  </div>
  <div
    className="copyright"
  >
    <p>
      Copyright Dobby 2022, All Rights Reserved.
    </p>
  </div>
</footer>
`;
